src/applications/personalization/profile360/actions/misc.js

```diff
@@ -82,7 +82,7 @@ function cleanAddressDataForUpdate(value) {
     addressLine2,
     addressLine3,
     addressPou,
-    addressType,
+    addressType: addressType || (countryName === 'United States' ? 'domestic' : 'international'),
```

Comment:
Yes this is janky and will improve this

Reply:
Is there any way this could also be a military address? (I've no context, so if it's a silly question, sorry!)